src/lib/auth/account.ts

```diff
@@ -65,16 +98,19 @@ const initializeFilesystem = async (fs: FileSystem): Promise<void> => {
   await fs.mkdir(webnative.path.directory(...ACCOUNT_SETTINGS_DIR))
 }
 
-export const loadAccount = async (username: string): Promise<void> => {
+export const loadAccount = async (hashedUsername: string, fullUsername: string): Promise<void> => {
   const session = await getStore(sessionStore).authStrategy.session()
 
   filesystemStore.set(session.fs)
 
   const backupStatus = await getBackupStatus(session.fs)
 
+  localStorage.setItem(USERNAME_STORAGE_KEY, fullUsername)
+
   sessionStore.update(state => ({
     ...state,
-    username,
+    hashedUsername,
+    username: fullUsername.split('#')[0],
```

Comment:
I'd make this:
```ts
{
  ...state,
  username: { hashed: "...", full: "..." }
}
```
instead, so the two never get out of sync.

Reply:
i like that, will update 👍🏼

Reply:
updated to be 
```
username: {
  full: '...',
  hashed: '...',
  trimmed: '...',
}
```